libevmasm/Assembly.h

```diff
@@ -222,6 +222,7 @@ class Assembly
 	{
 		uint8_t inputs = 0;
 		uint8_t outputs = 0;
+		bool canContinue = true;
```

Comment:
Maybe it would be clearer to call it `nonReturning` to closer match the EOF spec?

```suggestion
		bool nonReturning = false;
```

Reply:
I'd also add a comment clarifying that it's independent of the number of outputs. I.e. that for non-returning sections we still expect that to be there because it affects stack height calculation, even though it's not stored in the EOF header.

It would be best to mention it also in the docstring of `AbstractAssembly::registerFunction()`.

Reply:
I would have changed `_canContinue` in `AbstractAssembly` as well, so that in assembly we always deal only with `_nonReturning` and it's harder to mix them up. But I guess the way you did it is fine too. The struct was the more important one.

Reply:
Done.